components/SpotlightBanner/SpotlightBannerAside.tsx

```diff
@@ -1,7 +1,11 @@
 import React, { FunctionComponent } from 'react';
 
-const SpotlightBannerAside: FunctionComponent = ({ children }) => (
-  <div className="spotlight-banner__aside">{children}</div>
+interface SpotlightBannerAsideProps {
+  className: string;
+}
+
+const SpotlightBannerAside: FunctionComponent<SpotlightBannerAsideProps> = ({ children, className }) => (
+  <div className={className}>{children}</div>
```

Comment:
You need to add back the `spotlight-banner__aside` class ... any other classes can be added through the `className` prop

Reply:
Updated the classname